src/main/java/com/redhat/devtools/intellij/knative/actions/func/InvokeAction.java

```diff
@@ -64,11 +68,21 @@ public void actionPerformed(AnActionEvent anActionEvent, TreePath path, Object s
         if (dialog.isOK()) {
             ExecHelper.submit(() -> {
                 try {
-                    knCli.invokeFunc(model);
+                    String id = knCli.invokeFunc(model);
+                    Notification notification = new Notification(NOTIFICATION_ID,
+                            "Invoked successfully",
+                            "Function " + name + " has been successfully invoked with execution id " + id + " !",
```

Comment:
If by mistake id is not found in the JSON output, an empty string is returned thus we should handle this case to gererate proper information/notification to the user